test/integration/apiserver/apiserver_test.go

```diff
@@ -2934,6 +2934,112 @@ func TestDedupOwnerReferences(t *testing.T) {
 	}
 }
 
+func TestEnableEmulationVersion(t *testing.T) {
+	server := kubeapiservertesting.StartTestServerOrDie(t, nil, []string{"--emulation-version=1.22", "--feature-gates=EmulationVersion=true"}, framework.SharedEtcd())
+	defer server.TearDownFn()
+
+	rt, err := restclient.TransportFor(server.ClientConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tcs := []struct {
+		path               string
+		expectedStatusCode int
+	}{
+		{
+			path:               "/",
+			expectedStatusCode: 200,
+		},
+		{
+			path:               "/apis/apps/v1/deployments",
+			expectedStatusCode: 200,
+		},
+		{
+			path:               "/apis/flowcontrol.apiserver.k8s.io/v1beta1/flowschemas", // introduced at 1.20, removed at 1.26
+			expectedStatusCode: 200,
+		},
+		{
+			path:               "/apis/flowcontrol.apiserver.k8s.io/v1beta2/flowschemas", // introduced at 1.23, removed at 1.29
+			expectedStatusCode: 404,
+		},
+		{
+			path:               "/apis/flowcontrol.apiserver.k8s.io/v1beta3/flowschemas", // introduced at 1.26, removed at 1.32
+			expectedStatusCode: 404,
```

Comment:
Is the plan that we graduate this feature and don't need these tests _before_ these flow schemas APIs are fully deleted from code?  (does the timeline work out?)  If so this is fine, but let's add a comment explaining..

Reply:
We would need integration test even after this feature graduates. We should probably add a fake resource. Is it ok to add fake resource for integration tests?